modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,317 @@
+
+* The support for resource names to contain the dot character (".") is added.
+
+* The `hostAliases` array in the `PodTemplate` specification is added to the pod-level override of hostname resolution. It is achieved by modifying the `/etc/hosts` file.
+
+* A variable `$(tasks.status)` is introduced to access the aggregate execution status of tasks.
+
+* An entry-point binary build for Windows is added.
+
+
+[id="deprecated-features-1-5_{context}"]
+== Deprecated features
+
+* In the `when` expressions, support for fields written is PascalCase is removed. The `when` expressions only support fields written in lowercase.
++
+[NOTE]
+====
+If you had applied a pipeline with `when` expressions in Tekton Pipelines `v0.16` (Operator `v1.2.x`), you have to reapply it.
+====
+
+* When you upgrade the {pipelines-title} Operator to `v1.5`, the `openshift-client` and the `openshift-client-v-1-5-0` cluster tasks have the `SCRIPT` parameter. However, the `ARGS` parameter and the `git` resource are removed from the specification of the `openshift-client` cluster task. This is a breaking change, and only those cluster tasks that do not have a specific version in the `name` field of the `ClusterTask` resource upgrade seamlessly.
++
+To prevent the pipeline runs from breaking, use the `SCRIPT` parameter after the upgrade because it moves the values previously specified in the `ARGS` parameter into the `SCRIPT` parameter of the cluster task. For example:
++
+[source,yaml,subs="attributes+"]
+----
+...
+- name: deploy
+params:
+  - name: SCRIPT
+    value: oc rollout status <deployment-name>
+  runAfter:
+    - build
+  taskRef:
+    kind: ClusterTask
+    name: openshift-client
+...
+----
++
+
+* When you upgrade from {pipelines-title} Operator `v1.4` to `v1.5`, the profile names in which the `TektonConfig` custom resource is installed now change.
+
++
+.Profiles for `TektonConfig` custom resource
+[cols="1,1,1",options="header"]
+|===
+| Profiles in Pipelines 1.5 | Corresponding profile in Pipelines 1.4 | Installed Tekton components
+| All (_default profile_) | All (_default profile_) | Pipelines, Triggers, Add-ons
+| Basic | Default | Pipelines, Triggers
+| Lite | Basic | Pipelines
+
+|===
+
++
+[NOTE]
+====
+If you used `profile: all` in the `config` instance of the `TektonConfig` custom resource, no change is necessary in the resource specification.
+
+However, if the installed Operator is either in the Default or the Basic profile before the upgrade, you must edit the `config` instance of the `TektonConfig` custom resource after the upgrade. For example, if the configuration was `profile: basic` before the upgrade, ensure that it is `profile: lite` after upgrading to Pipelines 1.5.
+====
++
+
+* The `disable-home-env-overwrite` and `disable-working-dir-overwrite` fields are now deprecated and will be removed in a future release. For this release, the default value of these flags is set to `true` for backward compatibility.
```

Comment:
Good, here the future tense works well, because you still bring the users attention to the current release when you specifically say, `For this release, the default value...` the problem with the future is it distracts the user from what's going on now, so good job with redirecting users' attention to the current RN.